deploy/docker/cp-edge/sync-routes.py

```diff
@@ -671,14 +673,28 @@ def update_svc_url_for_run(run_id):
                 do_log('Asking for service url update for the run {}, but service_url_dict is empty for this run'.format(run_id))
 
 
+def find_preference(api_preference_query, preference_name):
+        load_method = os.path.join(api_url, api_preference_query)
+        response = call_api(load_method)
+        if response and "payload" in response and "name" in response["payload"] \
+                and response["payload"]["name"] == preference_name and "value" in response["payload"]:
+                return response["payload"]["value"]
+        return None
+
+
 do_log('============ Started iteration ============')
 
 kube_api = HTTPClient(KubeConfig.from_service_account())
 kube_api.session.verify = False
 
-edge_kube_service = Service.objects(kube_api).filter(selector={EDGE_SVC_ROLE_LABEL: EDGE_SVC_ROLE_LABEL_VALUE})
+default_edge_region = find_preference(API_GET_DEFAULT_EDGE_REGION_PREF, 'default.edge.region')
```

Comment:
lets add some warning here that we cant find default edge region, because if `default_edge_region` == None we will never create dns record for any of edge server -> will never to be able to use such link -> it is good to have at least warning in EDGE logs about it and may be we need to log it in run logs as well